tests/lib/validator_test.py

```diff
@@ -640,6 +640,7 @@ def test_mac_restriction_in_current_mac_in_desire(self):
         iface_state[Interface.MAC] = MAC0
         bond_config = iface_state[Bond.CONFIG_SUBTREE]
         bond_config.pop(Bond.MODE)
+        bond_config.pop(Bond.OPTIONS_SUBTREE)
```

Comment:
Why this is included?

Reply:
This is part of `state: An empty dict value is treated as an overwrite`. The patch is needed because the unit tests are using the empty dict as an overwrite. I could remove the change from this patch but I would prefer a complete backport rather than a modified backport.

Thanks!